src/App.tsx

```diff
@@ -398,7 +398,7 @@ export const App = ({
 						)}
 					</div>
 				) : (
-					<React.Fragment>
```

Comment:
Worth updating the PR description to remove references to this being needed once it all goes through